website/i18n/en.json

```diff
@@ -437,7 +437,8 @@
     "This project is used by all these people|no description given": "This project is used by all these people",
     "More|no description given": "More",
     "Users|no description given": "Users",
-    "This project is used by many folks|no description given": "This project is used by many folks",
+    "Verdaccio is used by many open source projects...|no description given": "Verdaccio is used by many open source projects...",
```

Comment:
`Verdaccio is used by many open source projects...` Since in the layout we cannot see differences between OSS and companies, perhaps we should rethink this line.

Reply:
Maybe we can differentiate between users who are *Sponsors* and others?🙂

Reply:
Yes, that would be great !

Reply:
In some point ... hopefully, 🤞 🤞 🤞 🤞 🤞 🤞 🤞  some soul would want to support us so we need a special space for them 😋

Reply:
One thing is, if you see the front page + scroll down ...

![Screen Shot 2019-05-12 at 7 20 40 PM](https://user-images.githubusercontent.com/558752/57585637-13cf4c80-74eb-11e9-8c64-909d04ae221c.png)

Neutrino should not be there, that space should be for those companies that support us such the case of SheetJS. pnpm was representing the OSS project which would be great to have some (limited to 3 o 4) in the front page, but separated from sponsors.